homeassistant/components/bang_olufsen/strings.json

```diff
@@ -1,19 +1,17 @@
 {
   "config": {
-    "abort": {
-      "already_configured": "[%key:common::config_flow::abort::single_instance_allowed%]",
-      "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
+    "error": {
       "api_exception": "[%key:common::config_flow::error::invalid_host%]",
       "client_connector_error": "[%key:common::config_flow::error::invalid_host%]",
       "timeout_error": "[%key:common::config_flow::error::timeout_connect%]",
       "value_error": "[%key:common::config_flow::error::invalid_host%]"
```

Comment:
Since the API only allows actual IP addresses, maybe this integration should have it's own error message? Or is "Invalid hostname or IP address" good enough?

Reply:
Yes, I've changed the error message to be "Invalid IPv4 address" and the last commit added IPv4 address checking